Overview.bs

```diff
@@ -92,7 +92,7 @@ Overview {#patch-overview}
 In the patch subset approach to incremental font transfer a server generates binary patches which a
 client applies to a subset of the font in order to extend the coverage of that font subset. The server
 is stateless, it does not maintain any session data for clients between requests. Thus when a client
-requests the generation of a patch from the server it must fully describe the current subset of the
+requests the generation of a patch from the server it has to fully describe the current subset of the
```

Comment:
I don't understand the change from "must" to "has to".

Reply:
The intention of this statement as originally written wasn't to make a normative requirement. It's just meant to be an explanation of the consequences of the stateless design. The requirements around what the client must send are covered further on in the specific sections about client to server requests. Just noticed this while updating the subset section.